qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/10_basic.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 "Index data, search, and create things in the cluster state that we'll validate are there after the ugprade":
   - skip:
+      version: " - 5.5.99"
```

Comment:
I think just adding this here will break the tests that run after it. If we push too hard down this road I think we'll end up skipping these tests almost entirely for versions before 5.6.0 and we don't want that.

I wonder if instead we can split the search template part into another test. We probably rig it so that we run one test before 5.6.0 that doesn't expect the warning and one >= 5.6.0 that expects the warning.

Reply:
I am with @nik9000 here.